src/common/Input/Input.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+interface inputProps {
+	placeholder: string;
+	value: string;
+	name?: string;
+	label?: string;
+	onChange: (e: React.ChangeEvent) => void;
+}
+
+export default function Input(props: inputProps): JSX.Element {
+	const { placeholder, value, name = '', label = '', onChange } = props;
+	const element: unknown =
```

Comment:
In this case defining a type is unnecessary because Typescript is able to identify based on the ternary operator
```suggestion
	const element =
```